### PR TITLE
Connected "pods" with "Using kubectl to Create a Deployment" tutorial.

### PR DESCRIPTION
**Pull request by T-Lakshmi**
Fixes: #43953

### CODE REVIEW

**Comment by T-Lakshmi** [comment]
[Preview page](https://deploy-preview-44051--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)

@tengqm, @onlydole,
PTAL!

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 1]
```diff
@@ -130,7 +130,7 @@ <h3>Deploy an app</h3>
         <div class="row">
             <div class="col-md-12">
                 <h3>View the app</h3>
-                <p>Pods that are running inside Kubernetes are running on a private, isolated network.
+                <p><a href="/docs/concepts/workloads/pods/">Pods</a> that are running inside Kubernetes are running on a private, isolated network.
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:133]: This helps; I would also like to tell the reader that right now we're not explaining what Pods are in any detail (so they know it's OK to continue, confident they'll learn later).

Comment (T-Lakshmi) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:133]: @sftim,
Thanks for the review, updated the text.

**Review by sftim — COMMENTED** [review]
Thanks
/lgtm

**Comment by T-Lakshmi** [comment]
Can anyone please approve this PR to get merge soon.

**Comment by kundan2707** [comment]
> Can anyone please approve this PR to get merge soon.

squash all of your commit to single one then only it can be approved.

**Comment by T-Lakshmi** [comment]
> @T-Lakshmi, please ignore that advice - I'm afraid it's not true
> We would still _prefer_ that you squash the commits, though.

IMK, If PR have multiple commits and we add label `tide/merge-method-squash`, commits will squash automatically while PR merge. We don't need to squash these explicitly. But, we generally squash it for simplicity if PR has multiple commits.

**Comment by T-Lakshmi** [comment]
@sftim, @kundan2707,
I did not squash the commits because the label `tide/merge-method-squash` was added and there were only two commits in the PR.
However, I have squashed the commits as per your suggestions.
Thanks,

**Comment by T-Lakshmi** [comment]
/assign @annajung,

Hi @annajung,
Requesting for PR approval.
PTAL!!

**Comment by sftim** [comment]
I can approve this.
/approve

**Pull request merged**